TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift

```diff
@@ -32,21 +32,24 @@ public protocol FactorPayload {
   var factorType: FactorType { get }
 }
 
-///Describes the information required to create a **Factor** which type is **Push**.
+/// Describes the information required to create a **Factor** which type is **Push**.
 public struct PushFactorPayload: FactorPayload {
-  ///A human readable description of this resource, up to 64 characters. For a push factor, this can be the device's name.
+  /// A human readable description of this resource, up to 64 characters. For a push factor, this can be the device's name.
   public let friendlyName: String
-  ///The unique SID identifier of the Service.
+  /// The unique SID identifier of the Service.
   public let serviceSid: String
-  ///Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
+  /// Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
   ///because the systems that will process this attribute assume it is not directly identifying information.
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/9cbd540c6135af3d1641ad17aef5ccc5e04d146c/TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift#L42"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>FactorPayload.swift:42</code></td>
    </tr>
  </tbody>
</table>